src/sass/_Fabric.Typography.Mixins.scss

```diff
@@ -5,7 +5,7 @@
 // --------------------------------------------------
 // Fabric Core Typography mixins
 
-@mixin ms-baseFont() {
+@mixin ms-baseFont {
   font-family: $ms-font-family-base;
```

Comment:
I'm not sure we want to set this just yet. What was the reason for including it here now?